client/state/sites/selectors.js

```diff
@@ -145,7 +170,7 @@ export function isJetpackMinimumVersion( state, siteId, version ) {
  * @return {?String}        Site slug
  */
 export function getSiteSlug( state, siteId ) {
-	const site = getSite( state, siteId );
+	const site = getRawSite( state, siteId );
```

Comment:
`getSite` uses `getSiteSlug`, and it would be less performant to calculate all the other options and then throw them away.